src/messages/en.json

```diff
@@ -57,8 +57,8 @@
 	"cognitive.parse.problem.databaseinfo": "I'm having problems understanding the name of your database. To display database information use *cloudant info database [database]*",
 	"cognitive.parse.problem.createdatabase": "I'm having problems understanding the name of the database. To create the database use *cloudant create database [database]*",
 	"cognitive.parse.problem.setpermissions.databasename": "I'm having problems understanding the name of your database. To set permissions to the database use *cloudant set permissions [database] [user]*",
+	"cognitive.parse.problem.setpermissions.username": "I'm having problems understanding the name of your user. To set permissions to the database use *cloudant set permissions [database] [user]*",
```

Comment:
I am not an expert here.  It sounds better to me to say "To set permissions **for** the database use *cloudant set permissions"